bigtop.bom

```diff
@@ -166,7 +166,7 @@ bigtop {
     'hive' {
       name    = 'hive'
       relNotes = 'Apache Hive'
-      version { base = '3.1.2'; pkg = base; release = 2 }
+      version { base = '3.1.2'; pkg = base; release = 3 }
```

Comment:
I think we don't need to increment release number since there is no release (i.e. 3.1.0 and 3.0.1) after https://github.com/apache/bigtop/pull/841. @yoda-mon.